classes/log/td.php

```diff
@@ -16,9 +16,9 @@ public static function _init(){
 	public static function write($level, $msg, $method = null)
 	{
 		$log_threshold = \Config::get('log_threshold');
-		$config = \Config::get('log',array());
+		$config = \Config::get('log', array());
 
-		if( isset($config['drivers']['td']['log_threshold']))
+		if (isset($config['drivers']['td']['log_threshold']))
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after closing parenthesis; found 9